### PR TITLE
Feat structure

### PR DESCRIPTION
**Pull request by zengqiangHe**
Hi there,

I have just:

- created a EventCard component
- installed Styled Component
- created a list with hardcoded details for the cards
- given the card component some simple style.

Thanks for the review !

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -1,22 +1,46 @@
...
+      location: 'Papenhuder Str. 67, Hamburg',
+    },
+    {
+      id: '3',
+      title: 'After Work Run an der Alster',
+      text: 'Am Dienstag haben wir endlich gutes Wetter in Hamburg! Wollen wir nach der Arbeit zusammen Joggen gehen?',
+      date_time: '11-08-2022 18 Uhr',
+      location: 'An der Alster, Hamburg',
+    },
+  ];
```
Comment (mpagels) [inline client/src/App.js:29]: I would recommend moving the part out of the `App.jsx`. Put it just before the line 6.

File: client/src/components/header/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import styled from "styled-components";
+
+const Header = () => {
+    return <StyledHeader>BockWurst</StyledHeader>;
+  };
```
Comment (mpagels) [inline client/src/components/header/Header.jsx:6]: You could pass a prop here, to make the `Header` reusable.
```suggestion
const Header = ({label}) => {
    return <StyledHeader>{label}</StyledHeader>;
  };
```

File: client/src/App.js [hunk 3]
```diff
@@ -1,22 +1,46 @@
...
 
   return (
     <div className="App">
-      <h1>Users</h1>
-      <button onClick={fetchSomething}>Fetch users</button>
-      <ul>
-        {users.map(user => (
-          <li key={user.id}>{user.name}</li>
+      <Header />
+      <ul style={{padding: '5%'}} role="list">
```
Comment (mpagels) [inline client/src/App.js:34]: Inline styling is not a good option. Why not using `style-components` here?

File: client/src/index.js [hunk 4]
```diff
@@ -1,11 +1,12 @@
...
 import ReactDOM from 'react-dom/client';
-import './index.css';
 import App from './App';
+import GlobalStyles from './pages/GlobalStyles';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
     <App />
+    <GlobalStyles />
```
Comment (mpagels) [inline client/src/index.js:10]: Not sure, but I would put `<GlobalStyles />` before `<App />`

**Review by mpagels — APPROVED** [review]
Hey Zach,
very good work here. I found some small issues. And I noticed that you already installed `React Router` but not using it in your app. I highly recommend just installing dependency, when it is part of the pull request. The same for empty files like `pages/create.jsx` and `pages/home.jsx`

File: client/src/components/header/Header.jsx (hunk 2 above)
Comment (zengqiangHe) [reply client/src/components/header/Header.jsx:6]: Done, thank you!

File: client/src/App.js (hunk 1 above)
Comment (zengqiangHe) [reply client/src/App.js:29]: Done, thank you!

File: client/src/index.js (hunk 4 above)
Comment (zengqiangHe) [reply client/src/index.js:10]: I just changed it, thanks!

File: client/src/App.js [hunk 5]
```diff
@@ -1,37 +1,38 @@
...
-      date_time: '11-08-2022 18 Uhr',
-      location: 'An der Alster, Hamburg',
-    },
-  ];
 
   return (
     <div className="App">
-      <Header />
-      <ul style={{padding: '5%'}} role="list">
+      <Header header="BockWurst"/>
```
Comment (mpagels) [inline client/src/App.js:34]: I would recommend renaming the props into `headerLabel` or just `label`.

**Review by mpagels — APPROVED** [review]
Awesome Zach. Have just one small thing. It is not necessary to change my issue. But I would recommend it. When you've done it, you can merge it. 

Approved.

**Comment by mpagels** [comment]
I just saw that you have `conflicts` with the branch. Just FYI! If you need help with that, feel free to ask me.

File: client/src/App.js (hunk 5 above)
Comment (zengqiangHe) [reply client/src/App.js:34]: Done!

**Pull request merged**